packages/models-library/src/models_library/settings/rabbit.py

```diff
@@ -27,9 +27,10 @@ class RabbitConfig(BaseSettings):
 
     # channels
     channels: Dict[str, str] = {
```

Comment:
here we should definitively use some ``TypedDict`` and replace the ``Dict[str, str]``